### PR TITLE
Add assets and pug file

### PR DESCRIPTION
**Pull request by Nezumi-2711**

### CODE REVIEW

File: practice/practice-02/src/nav-menu.pug [hunk 1]
```diff
@@ -0,0 +1,8 @@
+nav.nav
+  input.toggle-menu(type="checkbox")
+  .icon-menu
+  ul.nav-list
+    each val in ["Today Special Offers","Why FoodHut","Our Menu","Our Popular Food"]
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/nav-menu.pug]: Should we define variable for array?

File: practice/practice-02/src/nav-menu.pug [hunk 2]
```diff
@@ -0,0 +1,8 @@
+nav.nav
```
Comment (phuocduong-agilityio) [inline practice/practice-02/src/nav-menu.pug]: For all partial content like this, could we have a folder

File: practice/practice-02/src/nav-menu.pug (hunk 1 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/nav-menu.pug]: Fixed.

File: practice/practice-02/src/nav-menu.pug (hunk 2 above)
Comment (Nezumi-2711) [reply practice/practice-02/src/nav-menu.pug]: Fixed.

**Pull request merged**